app/src/main.c

```diff
@@ -0,0 +1,56 @@
+/**
+ * @author Teslabs Engineering S.L.
+ * @date 2021
+ * @copyright Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <drivers/sensor.h>
+
+#include "app_version.h"
+#include "foo.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(main, CONFIG_APP_LOG_LEVEL);
+
+int main(void)
+{
+	int ret;
+	const struct device *sensor;
+
+	printk("Zephyr Example Application %s\n", APP_VERSION_STR);
+
+	ret = app_foo_bar();
+	if (ret < 0) {
+		LOG_ERR("app_foo_bar failed (%d)", ret);
+		return ret;
+	}
+
+	sensor = DEVICE_DT_GET(DT_CHOSEN(sensor));
```

Comment:
Please just use DT_NODELABEL directly. Chosen nodes are more for system-wide settings. Most of the time a node label is the right thing, but people don't reach for it because it's confusing.